lib/infrastructure_collector.rb

```diff
@@ -88,7 +83,7 @@ def run
       properties.delete(:hostFolder)
       properties.delete(:network)
       properties.delete(:datastores) # Datastores from vSphere are referred to as 'volumes' in our domain
-
+      properties[:tags] = "vmware-collector"
```

Comment:
This text can go with single quotes since it doesn't have variable interpolation.